Makefile

```diff
@@ -302,18 +302,14 @@ endif
 # https://github.com/operator-framework/community-operators/blob/7f1438c/docs/packaging-operator.md#updating-your-existing-operator
 .PHONY: catalog-build
 catalog-build: opm ## Build a catalog image.
-	$(shell mkdir rsct-temp)
-	$(eval TMP_DIR := rsct-temp)
-	@envsubst < catalog/preamble_config_template.json > $(TMP_DIR)/rsct-operator-catalog.json
-	$(OPM) render $(BUNDLE_IMGS) >> $(TMP_DIR)/rsct-operator-catalog.json
-	$(OPM) generate dockerfile $(TMP_DIR) -i quay.io/operator-framework/opm:${OPM_VERSION}
+	$(OPM) render $(BUNDLE_IMGS) > catalog/released/$(VERSION).json
```

Comment:
right now `VERSION` is set to 0.0.1, how this will work user doesn't specify the version,? do we have any predefined way to run this target with different versions? anything documented somewhere?

>Note: its always good to point to the current release user doesn't specify the version

Reply:
While building the images for different `VERSION`, it can be set as an environment variable, and an entry needs to be added accordingly in [index.yaml](https://github.com/varad-ahirwadkar/rsct-operator/blob/catalog/catalog/index.yaml).

I will create a developer's documentation and will set the default `VERSION` to current release.